requirements.txt

```diff
@@ -0,0 +1,3 @@
+mock
+python-snap7~=0.11
```

Comment:
why add a requirements.txt file? and why add the library itself to it?

Reply:
Ops, that shouldn't have happen....